### PR TITLE
Improved sort section styling and added placeholder text for results(#27xhv5k)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by dt2patel** [comment]
Task linked: [CU-27xhv5k Improve stying of items of sort section and add placeholder text for results on the find screen](https://app.clickup.com/t/27xhv5k)

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/163970832-50965f7e-e36c-4949-8fd6-b2c68795e3db.mp4

File: src/views/FindOrder.vue [hunk 1]
```diff
@@ -44,20 +44,26 @@
...
+              <ion-item lines="none">
+                <ion-icon slot="start" :icon="documentTextOutline" />
+                <ion-label class="ion-text-wrap">{{ $t("Show order items") }}</ion-label>
+                <ion-toggle color="secondary" :checked="showOrderItems" @ionChange="() => showOrderItems = !showOrderItems"/>
+              </ion-item>
+
+              <ion-item lines="none">
+                <ion-icon slot="start" :icon="swapVerticalOutline" />
+                <ion-label class="ion-text-wrap">{{ $t("Sort") }}</ion-label>
+                <ion-select :value="sort" @ionChange="sortOrders($event.detail.value)">
```
Comment (dt2patel) [inline src/views/FindOrder.vue]: make popover inline.

**Review by dt2patel — CHANGES_REQUESTED** [review]
One change

File: src/views/FindOrder.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/FindOrder.vue]: This change is handled in the following PR:
https://github.com/hotwax/commerce-hub/pull/117

**Pull request merged**